src/tool_cfgable.h

```diff
@@ -337,7 +337,7 @@ struct GlobalConfig {
   unsigned short parallel_max; /* MAX_PARALLEL is the maximum */
   bool parallel_connect;
   char *help_category;            /* The help category, if set */
-  struct var *variables;
+  struct curl_var *variables;
```

Comment:
I would probably prefer `tool_var` or similar, simply because `curl_` is a prefix that we typically reserve for global and public symbols provided by libcurl.